unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlaceCard/PlaceCardComponentView.cs

```diff
@@ -1,12 +1,15 @@
 using DCL;
 using DCL.Helpers;
+using System;
 using System.Collections.Generic;
 using TMPro;
 using UnityEngine;
 using UnityEngine.UI;
 
 public interface IPlaceCardComponentView
 {
+    event Action<string, bool> OnFavoritePlaceChange;
```

Comment:
`public` modifier is missing

Reply:
why do we need the public modifier in the interface?

Reply:
ah sorry, my bad, didn't saw it is an interface 😅